evap/staff/templates/staff_semester_grade_reminder.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'staff_semester_base.html' %}
+
+{% load evaluation_filters %}
```

Comment:
you can probably remove that (in there are a few helper methods and you're using none of them)